src/placeos/api_wrapper/endpoint.cr

```diff
@@ -11,13 +11,22 @@ module PlaceOS
     def initialize(@client : APIWrapper)
     end
 
+    module Search(T)
+    end
+
```

Comment:
you might be able to  make this generic, but you'll probably need to call super for a few methods that need extra params